### ISSUE TITLE
Nightwatch Accessibility Scan for Content Build Improvements

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#24395]
## High Level User Story/ies
As a content build stakeholder, I need a human readable accessibility scan report so I can quickly triage the issues discovered and begin to resolve them.

## Hypothesis or Bet
If the accessibility scan reports are more easily triaged then accessibility issues will be more quickly resolved.

## OKR
508 Compliance Issues in the content build are mitigated

### What must be true in order for you to consider this epic complete? 

The accessibility scan generates a human readable report. Stakeholders are notified of the report's high level results via Slack. Content build stakeholders are satisfied with the report's contents.
